AISKU/Tests/applicationinsights.e2e.tests.ts

```diff
@@ -82,6 +85,8 @@ export class ApplicationInsightsTests extends TestClass {
             this.successSpy = this.sandbox.spy(sender, '_onSuccess');
             this.loggingSpy = this.sandbox.stub(this._ai['core'].logger, 'throwInternal');
             this.trackSpy = this.sandbox.spy(this._ai['dependencies'], 'trackDependencyDataInternal')
+            this.isSampledInSpy = this.sandbox.stub(sender, '_isSampledIn', () => true);
```

Comment:
I don't think this var is used anywhere, so you can just not assign this to a variable